### PR TITLE
feat: added rate limiting for TeleTan creation

### PR DESCRIPTION
**Pull request by f11h**

### CODE REVIEW

**Comment by mlaue-tech** [comment]
Planned for a later release (e.g. 1.1.0)

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -79,3 +83,5 @@ cwa-testresult-server:
     key-store-password: changeit
     trust-store: classpath:truststore.jks
     trust-store-password: changeit
+server:
```
Comment (dfischer-tech) [inline src/main/resources/application.yml:86]: really needed?

File: src/test/java/app/coronawarn/verification/VerificationApplicationTest.java [hunk 2]
```diff
@@ -24,16 +24,7 @@
...
-import app.coronawarn.verification.model.AuthorizationRole;
-import app.coronawarn.verification.model.HashedGuid;
-import app.coronawarn.verification.model.RegistrationToken;
-import app.coronawarn.verification.model.RegistrationTokenKeyType;
-import app.coronawarn.verification.model.RegistrationTokenRequest;
-import app.coronawarn.verification.model.Tan;
-import app.coronawarn.verification.model.TanSourceOfTrust;
-import app.coronawarn.verification.model.TanType;
-import app.coronawarn.verification.model.TestResult;
+import app.coronawarn.verification.model.*;
```
Comment (dfischer-tech) [inline src/test/java/app/coronawarn/verification/VerificationApplicationTest.java:27]: I recommend the import of classes and not an entire package

File: src/test/java/app/coronawarn/verification/VerificationApplicationTest.java [hunk 3]
```diff
@@ -75,6 +46,20 @@
...
 import org.springframework.test.context.junit4.SpringRunner;
 import org.springframework.test.web.servlet.MockMvc;
+
+import java.io.UnsupportedEncodingException;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.PrivateKey;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.util.*;
```
Comment (dfischer-tech) [inline src/test/java/app/coronawarn/verification/VerificationApplicationTest.java:56]: same like here

File: src/test/java/app/coronawarn/verification/service/TanServiceTest.java [hunk 4]
```diff
@@ -231,6 +241,87 @@ public void testTeleTANFormat() {
...
+    tanService.generateVerificationTeleTan();
+
+    assertThat(tanService.isTeleTanRateLimitNotExceeded()).isFalse();
+  }
+
+  @Test
+  public void testLogMessagesForRateLimits() throws NoSuchFieldException, IllegalAccessException {
+    Logger loggerMock = Mockito.mock(Logger.class);
+
+    // Usage of Relection API to "inject" mock for SLF4J Logger
```
Comment (dfischer-tech) [inline src/test/java/app/coronawarn/verification/service/TanServiceTest.java:299]: Java reflection is bad practice. Here we should come up with something more elegant

File: src/main/resources/application.yml [hunk 5]
```diff
@@ -29,7 +29,7 @@ springdoc:
     path: /api/swagger
 management:
   server:
-    port: 8081
+    port: 8086
```
Comment (MichaelPFrey) [inline src/main/resources/application.yml:32]: Why does a pull request for implementing a rate limit change a port number?

Comment (f11h) [reply src/main/resources/application.yml:32]: Hi @MichaelPFrey , thats a bad mistake. I've changed it for local testing purpose. This port-change will be rollbacked in my next commit.

File: src/main/resources/application.yml (hunk 1 above)
Comment (f11h) [reply src/main/resources/application.yml:86]: I have removed these 2 lines. I've added them for testing purpose and have forgotten to remove again

File: src/test/java/app/coronawarn/verification/VerificationApplicationTest.java (hunk 2 above)
Comment (f11h) [reply src/test/java/app/coronawarn/verification/VerificationApplicationTest.java:27]: I've rearranged the imports for this file

File: src/test/java/app/coronawarn/verification/service/TanServiceTest.java (hunk 4 above)
Comment (f11h) [reply src/test/java/app/coronawarn/verification/service/TanServiceTest.java:299]: As we have discussed in our team I have removed this test case.

File: src/test/java/app/coronawarn/verification/VerificationApplicationTest.java (hunk 3 above)
Comment (f11h) [reply src/test/java/app/coronawarn/verification/VerificationApplicationTest.java:56]: I've rearranged the imports for this file

**Pull request merged**